### DISCUSSION TITLE
HomeKit Bridge

### DISCUSSION THREAD

**Discussion by irlkitcom** [cryptk/haomnilogic-local#80] [Q&A]
I'm trying to add my pool and spa heaters to HomeKit but they are reporting as always on even when they are off, is there an easy solution to this?

**Comment by PelagicDev** [comment]
@irlkitcom Did you ever figure this out? TIA

**Comment by cryptk** [comment]
I don't have any Apple products in my house so I don't know how easily I'm going to be able to resolve this

**Reply by tony-rsrv** [reply]
The issue is with using Water Heaters in Home Assistant, HomeKit doesn't support Water Heaters natively so they are shown to HomeKit as thermostats, and the issue comes from the states. HomeKit only knows Off, Cool, Heat. In your integration, you have Off and On. I'm not too familiar with how everything works in Home Assistant, but I believe if you were able to change "On" to "Heat", this would resolve the issue. I have a HomeKit setup using your integration if you need any testing done.
